build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
For compatibility checks, the package manager relies on the product-semver property from system.properties, which is set to project.version.numeric by the build scripts. So either keep project.version.numeric or change the build scripts to use project.version (which then must be 3 digit).